components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -672,6 +682,9 @@
     }
 
     function updateBeanAndPostTo(postURL, data) {
+
+        encodeSPCertificate();
```

Comment:
After we finish the ajax call we can decode it right?

Reply:
Yes we can call the decode function. But it does not make any difference as we are redirecting to a new page or refresh the page in these cases. Adding it wont do any harm as well.

Reply:
Fixed